src/einsteinpy/tests/test_symbolic/test_tensor.py

```diff
@@ -147,3 +147,12 @@ def test_BaseRelativityTensor_automatic_calculation_of_free_variables():
             and (f in t2.functions)
             and (f in functions)
         )
+
+
+# tests fot Tensor Class to support scalars
+
+
+def test_tensor_scalar():
+    scalar = 11.89
+    scalar_tensor = Tensor(scalar)
```

Comment:
you would need to test for sympy expression type scalars also
```py
import pytest
from sympy.abc import y, z

# all code in between xD

@pytest.mark.parametrize("scalar",[11.89, y*z+5])
def test_tensor_scalar(scalar):
    scalar_tensor = Tensor(scalar)
    assert scalar_tensor.tensor().rank() == 0
```

BTW, it's a nice thing to learn. See this https://docs.pytest.org/en/latest/parametrize.html
pytest parametrize tests more than one input values for a single test function!!

Reply:
Thanks!!